docs/config.md

```diff
@@ -31,6 +31,12 @@ are shown below in ["Plugin settings"](#plugin-settings).
     the _Received: _header, ...
     Default is whatever Sys::Hostname's hostname() returns.
 
+- ar-me
```

Comment:
I'm not a big fan of the profusion of config files in djbware and projects inspired by it (like qpsmtpd). At least with djbware, the config file _names_ were indicative of what was in them. Joe sixpack doing a `ls config` will be hard pressed to intuit what the contents are. How about a name like `me-auth-results` or some such? At least then the `me` provides an indication (and logical grouping next to `me`) of the contents, and `auth-results` tells where it's used.